templates/templates.html

```diff
@@ -0,0 +1,22 @@
+<html>
```

Comment:
Do not integrate HTML with the back end at this point. that's for challenge four.